### PR TITLE
feat/get_post

### PR DESCRIPTION
**Pull request by spacjalex**

### CODE REVIEW

File: src/pkg/post/get_post.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	// getting params
+	postID := c.Query("postId")
+
+	if postID == "" {
+		c.AbortWithStatusJSON(http.StatusBadRequest, "Include postId in query params")
+		return
+	}
+
+	var post models.Post
+	if postID != "" {
```
Comment (nimaaza) [inline src/pkg/post/get_post.go:21]: This check here if postID != "" is unnecessary since it will always be true when this line is executed.

**Review by nimaaza — CHANGES_REQUESTED** [review]
With the suggested minor change this PR can be merged.

File: src/pkg/post/get_post.go (hunk 1 above)
Comment (spacjalex) [reply src/pkg/post/get_post.go]: Fixed :)

**Review by nimaaza — APPROVED** [review]
Hey @spacjalex, now the merge request can be closed. I will go ahead and do, squashing commits as you suggested with which I fully agree. Thank you 😉

**Pull request merged**